Pages/PendingVolunteersList.razor.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using Karma.Models;
+using Karma.Services;
+using MatBlazor;
+using Microsoft.AspNetCore.Components;
+
+namespace Karma.Pages
+{
+    public partial class PendingVolunteersList
+    {
+        [Inject]
+        public IDBServiceProvider m_DBServiceProvider { get; set; }
+
+        public string FilterValue = "";
```

Comment:
If it is not a property It should start with lowercase

Reply:
It starts from a lower case now.